src/components/TableToolbar.js

```diff
@@ -196,12 +198,18 @@ class TableToolbar extends React.Component {
     const { search, downloadCsv, print, viewColumns, filterTable } = options.textLabels.toolbar;
     const { showSearch, searchText } = this.state;
 
+    // prevent a customSearchRender component from calling the onSearchChange callback
+    let optionsCopy = Object.assign({}, options);
```

Comment:
I actually don't think we need this bit. Custom search render doesn't rely on `onSearchChange` at all, so we don't need to override here. It may also be a good thing to allow people to use this function in their own custom search renders if they want.

Reply:
Sounds good, just took it out.